config/processors/event_hub_log_security_azure.event_hub.conf

```diff
@@ -6,22 +6,73 @@ input {
   }
 }
 filter {
-  mutate {
-    add_field => {"log.source.hostname" => "eventhub_azure_security"}
-    add_field => { "event.module" => "eventhub azure security" }
-    add_field => { "event.dataset" => "security logs" }
-    add_field => { "observer.vendor" => "microsoft" }
-    add_field => { "observer.product" => "security logs" }
-    add_field => { "observer.type" => "eventhub" }
-  }
-  mutate {
-    add_field => { 
-      "event.module" => "Azure Security Logs"
-      }
-    copy => { "message" => "log.original" }
-  }
+    mutate {
+      add_field => {"log.source.hostname" => "eventhub_azure_security"}
+      add_field => { "event.module" => "eventhub azure security" }
+      add_field => { "event.dataset" => "security logs" }
+      add_field => { "observer.vendor" => "microsoft" }
+      add_field => { "observer.product" => "security logs" }
+      add_field => { "observer.type" => "eventhub" }
+    }
+	mutate {
+		copy => { "message" => "log.original" }
+	}
+	mutate {
+		gsub => ["message" ," ",""]
+    }
+	json {
+    	source => "message"
+    	target => "tmp"
+  	}
+
+	split {
+      	field => "[tmp][records]"
+	  	target => "tcs"
+    }
+
+	mutate{
```

Comment:
A space should proceed every {
Also please double check if indentations are correct.

Reply:
Updated it, thank you!

Reply:
Can you make indentations with 2 spaces please?